examples/getstarted/src/api/restaurant/content-types/restaurant/schema.json

```diff
@@ -10,6 +10,7 @@
   },
   "options": {
     "draftAndPublish": true,
+    "populateCreatorFields": true,
```

Comment:
Why do we need it all of a sudden?

Reply:
We don't I was doing a test I forgot I added it but I would actually like to keep it so we have one example in the get started

Reply:
Yup, agree on this one, nice addition!